k8s.go

```diff
@@ -388,3 +417,10 @@ func (c *Client) ReplaceKubernetesNode(clusterID, nodePoolID, nodeID string) (*h
 	h := rsp.Header()
 	return &h, validateResponse(rsp, http.StatusAccepted)
 }
+
+func (n *AutoScaling) Enabled() bool {
+	if n == nil || (n.MinNodeCount == nil || n.MaxNodeCount == nil) || (*n.MinNodeCount == 0 && *n.MaxNodeCount == 0) {
```

Comment:
Isn't the whole function identical to

```go
return !(
  n == nil ||
  n.MinNodeCount == nil ||
  n.MaxNodeCount == nil ||
  *n.MinNodeCount == 0 && *n.MaxNodeCount == 0)
```

?

Reply:
Moreover, you could just define the function without pointer receiver and skip the `n == nil` check.

Reply:
Also, why `n`?